osu.Game/Graphics/UserInterface/HoverSounds.cs

```diff
@@ -24,35 +25,39 @@ public class HoverSounds : CompositeDrawable
         /// <summary>
         /// Length of debounce for hover sound playback, in milliseconds. Default is 50ms.
         /// </summary>
-        public double HoverDebounceTime { get; } = 50;
+        public double HoverDebounceTime { get; } = 20;
 
         protected readonly HoverSampleSet SampleSet;
 
+        private Bindable<double> lastPlaybackTime;
+
         public HoverSounds(HoverSampleSet sampleSet = HoverSampleSet.Normal)
         {
             SampleSet = sampleSet;
             RelativeSizeAxes = Axes.Both;
         }
 
-        private ScheduledDelegate playDelegate;
+        [BackgroundDependencyLoader]
+        private void load(AudioManager audio, SessionStatics statics)
+        {
+            lastPlaybackTime = statics.GetBindable<double>(Static.LastHoverSoundPlaybackTime);
+
+            sampleHover = audio.Samples.Get($@"UI/generic-hover{SampleSet.GetDescription()}");
+        }
 
         protected override bool OnHover(HoverEvent e)
         {
-            playDelegate?.Cancel();
+            bool requiresDebounce = HoverDebounceTime <= 0;
+            bool enoughTimePassedSinceLastPlayback = lastPlaybackTime.Value == 0 || Time.Current - lastPlaybackTime.Value > HoverDebounceTime;
```

Comment:
checking 0 to determine if a playback has ever happened is sorta sketchy - `Bindable<double?>` could work better? don't really need `Precision` & friends with this one I don't think

Reply:
such an edge case i didn't bother. i originally had an inline comment saying i didn't bother. i guess i can try double? but not sure how well bindable handles that..

Reply:
`double?`'seems to work as expected, so let's go with that.